package.json

```diff
@@ -17,6 +17,17 @@
   },
   "homepage": "https://github.com/jannikschwarz/Drone-Stabilization_WebAPI#readme",
   "dependencies": {
-    "express": "^4.18.2"
+    "body-parser": "^1.20.1",
+    "express": "^4.18.2",
+    "firebase": "^9.13.0",
+    "morgan": "^1.10.0"
+  },
+  "devDependencies": {
+    "chai": "^4.3.6",
+    "eslint": "^8.26.0",
+    "mocha": "^10.1.0",
+    "mock-require": "^3.0.3",
+    "nodemon": "^2.0.20",
+    "sinon": "^14.0.1"
```

Comment:
You don't really need these except for nodemon 🤷 I don't think we have to do any unittests

Reply:
I will do them anyway =P